gulpfile.js

```diff
@@ -249,11 +256,11 @@ gulp.task('index', ['jsHint', 'scripts'], function() {
 
 // start local express server
 gulp.task('serve', function() {
-  express()
+  server = express()
     .use(!build ? connectLr() : function(){})
     .use(express.static(targetDir))
     .listen(port);
-  open('http://localhost:' + port + '/');
+  //open('http://localhost:' + port + '/');
```

Comment:
Hei Thomas,

Thanks for pointing out. Actually I was fiddling with different things to get it work. Sometimes just slipped out of my mind where I put `console.log` or commenting out code snippet for debugging.

BTW, just for my knowledge, I haven't fully understood why working on a branch is better in this case. Just trying to understand. If you have any concrete examples from your personal experience, it would be highly appreciated.

Actually my workflow was I forked the repository. I then worked on a branch called 'testing' and then I merged to the master of my repository. After that I created PR. 

Is there any pros and cons in this approach?

Thanks,